lib/rust/ensogl/core/src/timer/delayed_repeats.rs

```diff
@@ -0,0 +1,46 @@
+//! Delayed repeating timer based on browser timer APIs.
+
+use super::Interval;
+use super::Timeout;
+use crate::prelude::*;
+
+use std::time::Duration;
+
+/// A delayed repeating emitter based on `setTimeout` and `setIntevral` browser APIs.
+///
+/// The timer activation is controlled with `active` input.
+///
+/// When the timer becomes active, it will emit `on_trigger` event
+/// after initial `delay` duration, then further events will be emitted
+/// every `interval` duration.
```

Comment:
1. These docs are not formatted - some lines are longer, some are shorter.
2. These docs do not explain how to use it. The sentence `A delayed repeating emitter based on `setTimeout` and `setIntevral` browser APIs.` is probably correct, but a reader (like me) does not get any information how to use it. An example would be welcome.